docs/getting-started/installation/gcp.md

```diff
@@ -220,7 +220,7 @@ prefix.
   [here](https://cloud.google.com/sql/docs/postgres/connect-run#private-ip).
 - You should not store your `.env` file locally and build it into your Dockerfile. Ideally you should save your `.env`
   file in [Google Cloud Secret Manager](https://cloud.google.com/secret-manager) and in your CI/CD pipeline retrieve it
-  and add it to your conainter. Or, even better, let the container pick the `.env` up at runtime from Google Secret
+  and add it to your containter. Or, even better, let the container pick the `.env` up at runtime from Google Secret
```

Comment:
```suggestion
  and add it to your container. Or, even better, let the container pick the `.env` up at runtime from Google Secret
```

Let's correct the typo in this correction itself 😄

Reply:
:man_facepalming: Really sorry, early morning PR :sleeping: ... Thanks for merging !